src/admin-portal/create-campaign/Information.jsx

```diff
@@ -59,17 +60,17 @@ const Information = ({campaignDetails, setCampaignDetails, setStep, lists}) => {
         title: campaignDetails.title,
         tagline: campaignDetails.tagline,
         description: campaignDetails.description,
-        about_us_title: campaignDetails.about_us_title ,
+        about_us_title: campaignDetails.about_us_title,
```

Comment:
Can you make the `about_us_title` key optional?
If the about us title key is sent to the back-end, (that's not expecting it) it will complain

Reply:
i.e. If @abdullai-t  is spreading everything I send, yeah 🤣 🤣.... Is he? 🤣 

Wode3 lemme go and do it like that 🌚 🤣 just in case...